README.rst

```diff
@@ -71,3 +71,45 @@ You can filter tests based on the attributes. There is a attribute named ``test_
 For running ``webidentity_test`` you'll need have Keycloak running.
 
 In order to run any STS test you'll need to add "iam" section to the config file. For further reference on how your config file should look check ``s3tests.conf.SAMPLE``.
+
+========================
+ IAM policy tests
+========================
+
+This is a set of IAM policy tests. Tests use the Boto3 libraries.
+
+The tests use the Nose test framework. To get started, ensure you have
+the ``virtualenv`` software installed; e.g. on Debian/Ubuntu::
+
+	sudo apt-get install python-virtualenv
+
+and then run::
+
+	./bootstrap
+
+You will need to create a configuration file with the location of the
+service and two different credentials. A sample configuration file named
+``s3tests.conf.SAMPLE`` has been provided in this repo. This file can be
+used to run the iam policy tests on a Ceph cluster started with vstart.
+
+Note: These iam policy tests uses two users, "iam" and "s3 alt" as mentioned in s3tests.conf.SAMPLE.
```

Comment:
@ravindrachoudhari A lot of the information below and above this section is redundant. Could you have this section contain the just following:
1. What the IAM policy tests cover
2. How to filter out IAM policy tests
3. What a radosgw user needs to be configured with to run these tests
4. Which users in the example config file are already configured to run these tests from a Ceph cluster configured with vstart.sh

Reply:
Done. Please have a look.

Reply:
Looks good to me!